tests/integration/policy/testdata/ratelimitservice.yaml

```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 Istio Authors
```

Comment:
nit: I think we no longer need year in license now?

Reply:
Thanks..Done!